### PR TITLE
Fix Storageclass name and iterate over both RBD and FS

### PR DESCRIPTION
**Pull request by vasukulkarni**
Fix the testcase to use default storageclass and iterate over both RBD and FS

Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: tests/e2e/test_small_file_workload.py [hunk 1]
```diff
@@ -43,23 +40,32 @@ class TestSmallFileWorkload(E2ETest):
...
+    def test_smallfile_workload(self, ripsaw, file_size, files, threads, samples, interface):
         """
         Run SmallFile Workload
         """
         log.info("Apply Operator CRD")
         ripsaw.apply_crd('resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml')
-
-        log.info("Running SmallFile bench")
         sf_data = templating.load_yaml(constants.SMALLFILE_BENCHMARK_YAML)
+        if interface == constants.CEPHBLOCKPOOL:
```
Comment (RazTamir) [inline tests/e2e/test_small_file_workload.py]: Replacing if else:
```
storageclass = constants.DEFAULT_STORAGECLASS_RBD if interface == constants.CEPHBLOCKPOOL else constants.DEFAULT_STORAGECLASS_CEPHFS
log.info(f"Using {storageclass} Storageclass")
```

Comment (vasukulkarni) [reply tests/e2e/test_small_file_workload.py]: The line is 140 and flake8 complains

Comment (vasukulkarni) [reply tests/e2e/test_small_file_workload.py]: line too long (140 > 120 characters)cornflakes(E501)

File: tests/e2e/test_small_file_workload.py [hunk 2]
```diff
@@ -43,23 +40,31 @@ class TestSmallFileWorkload(E2ETest):
     """
 
     @pytest.mark.parametrize(
-        argnames=["file_size", "files", "threads", "samples"],
+        argnames=["file_size", "files", "threads", "samples", "interface"],
         argvalues=[
-            pytest.param(*[4, 50000, 4, 1], ),
-            pytest.param(*[16, 50000, 4, 1], ),
-            pytest.param(*[16, 200000, 4, 1], ),
+            pytest.param(*[4, 50000, 4, 1, constants.CEPHBLOCKPOOL], ),
```
Comment (ebenahar) [inline tests/e2e/test_small_file_workload.py]: missing polarion_ids

File: tests/e2e/test_small_file_workload.py (hunk 1 above)
Comment (vasukulkarni) [reply tests/e2e/test_small_file_workload.py]: I moved the log.info() out, but kept the if/else as is to limit < 80 chars to stop linter from complaining. I will resolve the comment for now.

File: tests/e2e/test_small_file_workload.py (hunk 2 above)
Comment (vasukulkarni) [reply tests/e2e/test_small_file_workload.py]: I will add to polarion and get the ids.

Comment (vasukulkarni) [reply tests/e2e/test_small_file_workload.py]: Fixed

**Comment by tiffanyn108** [comment]
LGTM

**Comment by vasukulkarni** [comment]
weird merge is disabled, checking settings.

**Comment by vasukulkarni** [comment]
thanks @petr-balogh

File: tests/e2e/test_small_file_workload.py [hunk 3]
```diff
@@ -100,7 +123,7 @@ def test_smallfile_workload(self, ripsaw, file_size, files, threads, samples):
             timeout=600
         )
         start_time = time.time()
-        timeout = 900
+        timeout = 1800
```
Comment (keesturam) [inline tests/e2e/test_small_file_workload.py]: @vasukulkarni why was the timeout from 900 to 1800 seconds?

**Pull request merged**